cordova-lib/src/plugman/uninstall.js

```diff
@@ -283,7 +283,7 @@ function runUninstallPlatform(actions, platform, project_dir, plugin_dir, plugin
             }
         };
 
-        var hooksRunner = new HooksRunner(projectRoot);
+        var hooksRunner = new HooksRunner(cordovaUtil.isCordova() || project_dir);
 
```

Comment:
why cordovaUtil.isCordova() || project_dir ?
wouldn't just using project_dir achieve the same result ?